plugins/basestate/BaseState.cc

```diff
@@ -99,6 +97,11 @@ class BaseState : public System,
         auto model = Model(_entity);
         this->m_baseLink = model.LinkByName(_ecm, baseLinkName);
```

Comment:
What if `baseLinkName` does not match any link in the model?

Reply:
Testing with a non existen link name I get the following error:

```
[Err] [EntityComponentManager.cc:1083] Trying to create a component of type [6822503852486540549] attached to entity [0], but this entity does not exist. This create component request will be ignored.
```

I will use a try block to verify that the link exists, otherwise raise an error with a log message.

Reply:
Solved in eb0ce408eb72439d69286e0305bcfec275b5d96d